common/config/src/main/java/io/helidon/common/config/EmptyConfig.java

```diff
@@ -139,6 +140,21 @@ public T get() throws ConfigException {
         public <N> ConfigValue<N> as(Function<T, N> mapper) {
             return new EmptyValue<>(key);
         }
+
+        @Override
+        public Supplier<T> supplier() {
+            return this::get;
+        }
+
+        @Override
+        public Supplier<T> supplier(T defaultValue) {
+            return () -> defaultValue;
+        }
+
+        @Override
+        public Supplier<Optional<T>> optionalSupplier() {
+            return this::asOptional;
```

Comment:
i didn't see a test on this with nulls, etc.

Reply:
Added a test and fixed the processor to correctly generate the config method